service_map.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "Service",
+  "name": "chs-delta-api",
+  "description": "A Companies House service which takes delta's fired from CHIPS and publishes them to a kafka topic",
```

Comment:
delta's -> deltas
kafka -> Kafka

Hmmm. Perhaps instead:
"Companies House RESTful service receiving a change ('delta') from CHIPS, publishing that delta to a Kafka topic; each type of delta (e.g. Shareholder) has its own RESTful endpoint mapped to its own topic."